### DISCUSSION TITLE
Lesson 3: Finishing the setup

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#1288] [Q&A]
while i  want to run " forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit" this command i got a error .Here is the error ''error: RPC failed; curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8)
error: 6053 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
fatal: clone of 'https://github.com/smartcontractkit/chainlink-brownie-contracts' into submodule path '/home/kingdevil/foundry-24/foundry-fund-me-24/lib/chainlink-brownie-contracts' failed
Error: 
git submodule exited with code 128"

**Comment by tobezhanabi** [comment]
if you face submodel 128 error, do this
- switch to a faster network and then 
- `git config --global http.postBuffer 524288000` and then
- `git gc --aggressive`
keep repeating it will work
